src/stream-net/IBatchOperations.cs

```diff
@@ -9,7 +9,8 @@ public interface IBatchOperations
         Task AddToMany(Activity activity, IEnumerable<string> feedIds);
         Task FollowMany(IEnumerable<Follow> follows, int activityCopyLimit = 300);
         Task<IEnumerable<Activity>> GetActivities(IEnumerable<string> ids = null, IEnumerable<ForeignIDTime> foreignIDTimes = null);
-        Task<IEnumerable<EnrichedActivity>> GetEnrichedFlatActivities(IEnumerable<string> ids = null, ReactionOption reactions = null, IEnumerable<ForeignIDTime> foreignIDTimes = null);
+        Task<IEnumerable<EnrichedActivity>> GetEnrichedFlatActivities(IEnumerable<string> ids, GetOptions options = null);

```

Comment:
this is for activities, not tied to feed types so I think it's better to drop flat keyword

Reply:
✅